### ISSUE TITLE
pip-gx issues on osx w/ brew python

### ISSUE THREAD

**Issue by dannon** [galaxyproject/galaxy#1640]
bx-python, MarkupSafe, PyYAML,SQLAlchemy, mercurial, numpy, pycrypto, Cheetah, pysam all attempt to build from source at current 16.01 or dev tips, assuming our pip can't figure out my platform correctly -- OSX, py2.7.11 from brew.

```
[yoplait@malleus:~/work/galaxy] python --version
Python 2.7.11
[yoplait@malleus:~/work/galaxy] which python  pysam_84_update
/usr/local/bin/python
[yoplait@malleus:~/work/galaxy] brew info python
python: stable 2.7.11 (bottled), HEAD
Interpreted, interactive, object-oriented programming language
https://www.python.org
/usr/local/Cellar/python/2.7.10_2 (6,194 files, 83.1M)
  Built from source
/usr/local/Cellar/python/2.7.11 (5,905 files, 80M) *
  Built from source
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/python.rb
```

And the weirdness:

```
PYTHONPATH= python -c 'from pip.pep425tags import get_supported; print "\n".join([str(x) for x in get_supported()])'
```

yields:

```
('cp27', 'cp27m', 'm')
('cp27', 'cp27m', 'a')
('cp27', 'cp27m', 'c')
('cp27', 'cp27m', 'o')
('cp27', 'cp27m', 's')
('cp27', 'cp27m', 'x')
('cp27', 'cp27m', '_')
('cp27', 'cp27m', '1')
('cp27', 'cp27m', '0')
('cp27', 'cp27m', '_')
('cp27', 'cp27m', '1')
('cp27', 'cp27m', '1')
('cp27', 'cp27m', '_')
('cp27', 'cp27m', 'x')
('cp27', 'cp27m', '8')
('cp27', 'cp27m', '6')
('cp27', 'cp27m', '_')
('cp27', 'cp27m', '6')
('cp27', 'cp27m', '4')
('cp27', 'none', 'm')
('cp27', 'none', 'a')
('cp27', 'none', 'c')
('cp27', 'none', 'o')
('cp27', 'none', 's')
('cp27', 'none', 'x')
('cp27', 'none', '_')
('cp27', 'none', '1')
('cp27', 'none', '0')
('cp27', 'none', '_')
('cp27', 'none', '1')
('cp27', 'none', '1')
('cp27', 'none', '_')
('cp27', 'none', 'x')
('cp27', 'none', '8')
('cp27', 'none', '6')
('cp27', 'none', '_')
('cp27', 'none', '6')
('cp27', 'none', '4')
('py2', 'none', '4')
('cp27', 'none', 'any')
('cp2', 'none', 'any')
('cp26', 'none', 'any')
('cp25', 'none', 'any')
('cp24', 'none', 'any')
('cp23', 'none', 'any')
('cp22', 'none', 'any')
('cp21', 'none', 'any')
('cp20', 'none', 'any')
('py27', 'none', 'any')
('py2', 'none', 'any')
('py26', 'none', 'any')
('py25', 'none', 'any')
('py24', 'none', 'any')
('py23', 'none', 'any')
('py22', 'none', 'any')
('py21', 'none', 'any')
('py20', 'none', 'any')
```

ping @natefoo any more details you need?

**Comment by natefoo** [comment]
That should be good, I'll get to this ASAP.

**Comment by dannon** [comment]
https://github.com/natefoo/pip/pull/1 should fix this. (works for me)

**Comment by natefoo** [comment]
Looks good. With this, it fetches the intel wheels?

**Comment by dannon** [comment]
Yep.

**Comment by dannon** [comment]
Closed with #1652
